### PR TITLE
[skip ci]update python example

### PR DESCRIPTION
**Pull request by wxyucs**

### CODE REVIEW

**Review by jiaweizone — APPROVED** [review]
LGTM

File: .gitignore [hunk 1]
```diff
@@ -179,3 +179,4 @@ data/
 /coverage_ut.info
 /lcov_cobertura.py
 /testresult/
+*.index
```
Comment (LHT129) [inline .gitignore]: empty line

**Review by LHT129 — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (wxyucs) [reply .gitignore]: fixed

**Review by LHT129 — APPROVED** [review]
LGTM

**Pull request merged**